client/src/App.tsx

```diff
@@ -3,16 +3,22 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import Login from "./pages/Login";
 import Register from "./pages/Register";
 import HomePage from "./pages/Homepage";
+import Navbar from "./components/navbar/Navbar";
+
+import './App.scss';
 
 const App = () => {
   return (
     <Router>
-      <Routes>
-        <Route path="/login" element={<Login />}/>
-        <Route path="/register" element={<Register />}/>
-        <Route path="/home" element={<HomePage />}/>
-        <Route path="*" element={<HomePage />}/>
-      </Routes>
+      <Navbar />
+      <div className="main-container">
+        <Routes>
+          <Route path="/login" element={<Login />} />
```

Comment:
Just a suggestion:
When create PRs trying to check before if there is changes that are meaningless like adding spaces because it adds clutter to the PR and make it abit harder to go through (If you indented to add spaces here thats fine)